express/scripts/utils.js

```diff
@@ -215,6 +216,7 @@ export function listenMiloEvents() {
 }
 
 export function decorateArea(area = document) {
+  autoUpdateContent(area, getLibs());
```

Comment:
This is a clean approach, but I need the most performant approach possible. 
That means instead of importing and delaying the whole page load by another 100-200 milliseconds just add the code from autoUpdateContent that you really really need even if it's copy pasted. Don't add the whole file if possible. Just the code that you really need executed here.